docs/non_containerized_NSFS.md

```diff
@@ -268,7 +268,7 @@ NSFS management CLI command will create both account and bucket dir if it's miss
 
 Non containerized NSFS certificate location is configured in system.json file under the property `nsfs_ssl_cert_dir` and the path should contain SSL files tls.key and tls.crt. System will use a cert from this dir to create a valid HTTPS connection. If cert is missing in this dir a self-signed SSL certificate will be generated. Make sure the path mentioned in `nsfs_ssl_cert_dir` is valid before running nsfs command, If the path is invalid then cert flow will fail.
 
-Non containerized NSFS allow nonsecure HTTP connection only when `allow_http` in system.json is true.
+Non containerized NSFS restrict insecure HTTP connections when `allow_http` is set to false in system.json. This is not the default behaviour.
```

Comment:
can you please add some instructions about how to fetch the stats and the output etc?